packages/expo-optimize/src/assets.ts

```diff
@@ -140,18 +136,31 @@ function filterImages(files: string[], projectRoot: string) {
 }
 
 // Calculate SHA256 Checksum value of a file based on its contents
-function calculateHash(filePath: string): string {
-  const contents = readFileSync(filePath);
+async function calculateHash(filePath: string): Promise<string> {
+  const contents = await new Promise<Buffer>((resolve, reject) =>
```

Comment:
generally we use `await fs.promises.readFile()` since Node.js was dropped earlier this year.